libs/design/src/molecules/accordion/accordion/accordion.component.html

```diff
@@ -1 +1 @@
-<ng-content select="daff-accordion-item"></ng-content>
\ No newline at end of file
+<ng-content></ng-content>
```

Comment:
I think this is probably fine but I wonder if @xelaint has thoughts about the implications of this change.

Reply:
I removed this change from this PR